docs/3.0rc/guides/deferred-tasks.mdx

```diff
@@ -0,0 +1,307 @@
+
+## Using deferred tasks
+
+Prefect tasks are Python functions that can be run immediately or submitted for background execution.
+
+You define a task by adding the `@task` decorator to a Python function, after which you can use the `delay` method to run the task in the background.
+
+If you submit the task for background execution, you'll run a task server in a separate process or container to execute the task. This process is similar to how you would run a Celery worker or an arq worker to execute background tasks.
+
+### Defining a task
+
+Add the `@task` decorator to a Python function to define a Prefect task:
+
+```python
+from prefect import task
+
+@task
+def my_background_task(name: str):
+    # Task logic here
+    print(f"Hello, {name}!")
+```
+
+### Calling tasks
+
+You can call a task to run it immediately, or you can defer the task by scheduling it for background execution with `Task.delay`.
+
+**NOTE**: It is also possible to submit tasks to a _task runner_ such as Ray or Dask -- and to defer task execution -- within a workflow, which in Prefect is called a _flow_. However, this document will focus on deferring task execution outside of workflows. For example, by calling `my_task.delay()` within a web application.
+
+However you run a task, Prefect will use your task configuration to manage and control task execution.
+The following example shows both methods mentioned earlier, calling a task and using `delay`:
+
+```python
+# Import the previously-defined task
+from my_tasks import my_background_task
+
+# Run the task immediately
+my_background_task("Joaquim")
+
+# Schedule the task for execution outside of this process
+my_background_task.delay("Agrajag")
+```
+
+For documentation on the features available for tasks, refer to the [Prefect Tasks documentation](https://docs-3.prefect.io/3.0rc/tutorial/tasks).
+
+### Executing deferred tasks with a task server
+
+To run tasks in a separate process or container, start a task server.
+
+The task server will continually receive deferred tasks to execute from Prefect's API, execute them, and report the results back to the API.
+
+**NOTE:** Task servers only run deferred tasks, not tasks you call directly.
+
+You can run a task server by passing tasks into the `prefect.task_server.serve()` method:
+
+```python tasks.py
+from prefect import task
+from prefect.task_server import serve
+
+
+@task
+def my_background_task(name: str):
+    # Task logic here
+    print(f"Hello, {name}!")
+
+
+if __name__ == "__main__":
+    # NOTE: The serve() function accepts multiple tasks. The Task Server 
+    # will listen for submitted task runs for all tasks passed in.
+    serve(my_background_task)
+```
+
+Run this script to start the task server.
+
+The task server should begin listening for submitted tasks. If tasks were submitted before the task server started, it will begin processing them.
```

Comment:
The word "submitted" could potentially be confusing here. We probably reserve that for submission to task runners.

Reply:
good flag, let me change